### PR TITLE
fix ceres-solver-1.13.0 test code error at mingw.

### PR DESCRIPTION
**Pull request by bygreencn**
fix ceres-solver-1.13.0 test code error at mingw.

### CODE REVIEW

File: mingw-w64-ceres-solver/PKGBUILD [hunk 1]
```diff
@@ -15,13 +15,20 @@ depends=("${MINGW_PACKAGE_PREFIX}-eigen3"
          "${MINGW_PACKAGE_PREFIX}-suitesparse")
 checkdepends=("${MINGW_PACKAGE_PREFIX}-gflags")
 source=("${url}${_realname}-${pkgver}.tar.gz"
-        "${_realname}-1.12.0.patch")
+        "${_realname}-1.13.0-CMakefiles.patch"
+        "${_realname}-1.13.0-fix-testcode-failed.patch")
+        
+#sha256sums=('745bfed55111e086954126b748eb9efe20e30be5b825c6dec3c525cf20afc895'
```
Comment (mati865) [inline mingw-w64-ceres-solver/PKGBUILD]: Remove old checksums.

Comment (bygreencn) [reply mingw-w64-ceres-solver/PKGBUILD]: Not need, since the early package may failed when compiling.

Comment (Alexpux) [reply mingw-w64-ceres-solver/PKGBUILD]: Check version in repo to decide increment or not

Comment (bygreencn) [reply mingw-w64-ceres-solver/PKGBUILD]: No special, just following the expression of 1.12.0.

Comment (filnet) [reply mingw-w64-ceres-solver/PKGBUILD]: Version in repo is 1.13.0-1 so the pkgrel needs to be increased.

$ pacman -Ss ceres
mingw32/mingw-w64-i686-ceres-solver 1.13.0-1
    A large scale non-linear optimization library (mingw-w64)
mingw64/mingw-w64-x86_64-ceres-solver 1.13.0-1
    A large scale non-linear optimization library (mingw-w64)

Comment (bygreencn) [reply mingw-w64-ceres-solver/PKGBUILD]: Got it，it is ok.

**Pull request merged**